README.md

```diff
@@ -196,6 +196,48 @@ Options:
 Examples:
   shep logs                 Launch an interactive CLI
   shep logs production foo  Shows logs for the `foo` function in the production environment
+
+```
+#### `shep config list`
+```
+shep config list [env] [functionName]
+
+Options:
+  --version  Show version number                                                                               [boolean]
+  --help     Show help                                                                                         [boolean]
+
+Examples:
+  shep config list            Launches an interactive CLI to choose env and functionName
+  shep config list beta foo   Lists the environment variables and values for foo with alias beta
```

Comment:
wait, I did use FOO. I am going to change this to the fn name. I still have it working that way